scripts/profiler/plot_nordic.py

```diff
@@ -391,63 +384,37 @@ def resize_event(event):
         plt.tight_layout()
 
     def close_event(self, event):
```

Comment:
This function is automatically executed because it's added as an event handler in line 164: `fig.canvas.mpl_connect('close_event', self.close_event)` and therefore it has to have fixed number of arguments.